pkg/util/sort_uniq.go

```diff
@@ -37,3 +37,31 @@ func uniqSorted(elements []string) []string {
 	}
 	return elements[:j+1]
 }
+
+// RemoveDuplicatesAndSort sorts and removes duplicates from a slice without doing it in place.
+func RemoveDuplicatesAndSort(elements []string) []string {
+	// isolate unique elements
+	found := make(map[string]bool)
+	unique := []string{}
+
+	for v := range elements {
+		if !found[elements[v]] {
+			unique = append(unique, elements[v])
+			found[elements[v]] = true
+		}
+	}
+
+	// sort the array
+	sort.Strings(unique)
```

Comment:
It would have been more efficient to remove the duplicates after the sorting.
It would remove the need to have a `found` map as you would know that duplicates are necessarily consecutive.

Reply:
Indeed ! It would've been similar to existing functions. I wanted to leverage them but couldn't do it in two places because of the circular dependency. Normally it should be fixed in the new commit